src/recipe.cpp

```diff
@@ -1839,15 +1839,22 @@ void Recipe::recalcOgFg()
       // being updated for no good reason.
       // NOTE: We are we recalculating all of these on load? Shouldn't we be
       // reading these values from the database somehow?
-      set( "og", "og", _og, false );
+      // NOTE: We always calculate this, so lets not cause
+      // a database dirty when loading recipes. This fixes
+      // starting the app and then closing causing a Save Changes.
+      // set( "og", "og", _og, false );
```

Comment:
Unless someone calls `Recipe::setOg()`, this will cause the og to _never_ get written to the database.